### DISCUSSION TITLE
ModuleNotFoundError: No module named 'gpiozero'

### DISCUSSION THREAD

**Discussion by maartenbaeten509** [gpiozero/gpiozero#1021] [Q&A]
I tried to use gpiozero on my raspberry pi 4, when i try to run the code it doesn't find the module. Anny suggestions? I tried downloading it by several different ways as stated on the gpiozero documentation page. Don't know what to do now.

**Comment by bennuttall** [comment]
Are you using the Raspberry PI OS, or another operating system? Which version of the OS are you using?

Are you using Python 3? Which specific version?

Is there anything unusual about your setup?

Can you show us what happens in the terminal if you run `sudo apt install python3-gpiozero`?
